openapi/index.yaml

```diff
@@ -338,25 +338,181 @@ paths:
         Note that this is an optimization for those services that need to send very high
         volumes of messages per day, to avoid having to make two API calls for each message.
       responses:
-        '200':
+        "200":
           description: Found.
           schema:
             $ref: "#/definitions/SubscriptionsFeed"
           examples:
             application/json:
               dateUTC: "2019-09-03"
-              subscriptions: [ "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855" ]
-              unsubscriptions: [ ]
-        '401':
+              subscriptions:
+                [
+                  "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
+                ]
+              unsubscriptions: []
+        "401":
           description: Unauthorized
-        '404':
+        "404":
           description: Subscriptions feed not available yet.
           schema:
             $ref: "#/definitions/ProblemJson"
-        '429':
+        "429":
           description: Too many requests
       parameters:
-      - $ref: "#/parameters/DateUTC"
+        - $ref: "#/parameters/DateUTC"
+  "/services":
+    post:
+      operationId: createService
+      summary: Create Service
+      description: |
+        Create a new Service with the attributes provided in the request payload.
+      parameters:
+        - in: body
+          name: body
+          description: The service payload.
+          schema:
+            "$ref": "#/definitions/ServicePayload"
+      responses:
+        "200":
+          description: Service created.
+          schema:
+            "$ref": "#/definitions/ServiceWithSubscriptionKeys"
+        "400":
+          description: Invalid payload.
+          schema:
+            $ref: "#/definitions/ProblemJson"
+        "401":
+          description: Unauthorized.
+        "429":
+          description: Too many requests.
+        "500":
+          description: Internal Server Error.
+    get:
+      operationId: getServices
+      summary: Get Services
+      description: |
+        Retrieve a list of services.
+      responses:
+        "200":
+          description: The list of service ids.
+          schema:
+            "$ref": "#/definitions/ServiceIdCollection"
+        "401":
+          description: Unauthorized.
+        "429":
+          description: Too many requests.
+  "/services/{service_id}":
+    parameters:
+      - name: service_id
+        in: path
+        type: string
+        required: true
+        description: The ID of an existing Service.
+    get:
+      operationId: getService
+      summary: Get Service
+      description: |
+        Retrieve a previously created service with the provided service ID.
+      responses:
+        "200":
+          description: Service found.
+          schema:
+            "$ref": "#/definitions/ServiceWithSubscriptionKeys"
+        "401":
```

Comment:
I see all the endpoint may return 401 (in case the api key is invalid i guess) but shouldn't they return 403 as well in case the calling user is not the _ower_ of the service that is being queried (it's not between this user subscriptions)

Reply:
Fixed in [d1fea39](https://github.com/pagopa/io-functions-services/pull/67/commits/d1fea39496f18158848e087c920b5e5e6c3c0073)

Reply:
it looks like there's a missing 403 on get